source/docs/api-reference/api-usage/migration-guide/control-requests-guide.rst

```diff
@@ -0,0 +1,207 @@
+
+.. tab-set::
+
+   .. tab-item:: Java
+      :sync: Java
+
+      .. code-block:: java
+
+         // class member variables
+         DutyCycle m_request = new DutyCycle(0);
+
+         // robot init, set m_follower to follow m_leader
+         // m_follower should NOT oppose leader
+         m_follower.setControl(new Follower(m_leader.getDeviceID(), false));
+         // set m_strictFollower to strict-follow m_leader
+         // strict followers ignore the leader's invert and use their own
+         m_strictFollower.setControl(new StrictFollower(m_leader.getDeviceID()));
+
+         // main robot code, command 100% output for m_leader
+         m_motor.setControl(m_request.withOutput(1.0));
+         // - m_follower and m_strictFollower will also run at 100% output
+         // - m_follower will follow m_leader's invert, while m_strictFollower
+         //   ignores it and uses its own
+
+   .. tab-item:: C++
+      :sync: C++
+
+      .. code-block:: cpp
+
+         // class member variables
+         controls::DutyCycle m_request{0};
+
+         // robot init, set m_follower to follow m_leader
+         // m_follower should NOT oppose leader
+         m_follower.SetControl(controls::Follower{m_leader.GetDeviceID(), false});
+         // set m_strictFollower to strict-follow m_leader
+         // strict followers ignore the leader's invert and use their own
+         m_strictFollower.SetControl(controls::StrictFollower{m_leader.GetDeviceID()});
+
+         // main robot code, command 100% output for m_leader
+         m_motor.SetControl(m_request.WithOutput(1.0));
+         // - m_follower and m_strictFollower will also run at 100% output
+         // - m_follower will follow m_leader's invert, while m_strictFollower
+         //   ignores it and uses its own
+
+Control Types
+-------------
+
+In Phoenix Pro, voltage compensation has been replaced with the ability to directly specify the :ref:`control output type <docs/api-reference/api-usage/device-specific/talonfx/talonfx-control-intro:control output types>`.
+
+All control output types are supported in open-loop and closed-loop control requests.
+
+.. list-table:: Open-loop Control Requests
+   :header-rows: 1
+
+   * - Phoenix 5
+     - Phoenix Pro
+
+   * - PercentOutput
+     - DutyCycleOut
+
+   * - PercentOutput + Voltage Compensation
+     - VoltageOut
+
+   * - Phoenix 5 does not support torque control
+     - TorqueCurrentFOC
+
+   * - Current closed-loop
+     - This has been deprecated in Phoenix Pro.
+
+       - Users looking to control torque should use TorqueCurrentFOC
+       - Users looking to limit current should use supply and stator current limits
+
+.. list-table:: Closed-loop Control Requests
+   :header-rows: 1
+
+   * - Phoenix 5
+     - Phoenix Pro
+
+   * - Position
+     - PositionDutyCycle
+
+   * - Velocity
+     - VelocityDutyCycle
+
+   * - MotionMagic
+     - MotionMagicDutyCycle
+
+   * - Closed-loop + Voltage Compensation
+     - {ClosedLoop}Voltage
```

Comment:
I think it's more correct to encompass names to be replaced with arrow-brackets, so it should be `<ClosedLoop>Voltage`

Typically it's done this way for things like linux help docs.

Reply:
For now we'll leave it as curly braces, and potentially move them to using arrow brackets based on feedback